src/main/java/uk/gov/register/core/EntryLog.java

```diff
@@ -73,15 +73,21 @@ public RegisterProof getRegisterProof(int entryNo) {
     public EntryProof getEntryProof(int entryNumber, int totalEntries) {
         List<String> auditProof = backingStoreDriver.withVerifiableLog(verifiableLog ->
                 verifiableLog.auditProof(entryNumber, totalEntries)
-                        .stream().map(this::bytesToString).collect(Collectors.toList()));
+                        .stream()
+                        .map(this::bytesToString)
+                        .map(p -> HashingAlgorithm.SHA256.toString() + ":" + p)
```

Comment:
You should not need to do this here if you change the `EntryProof.auditPath` property `List<HashValue>`. One purpose of creating the `HashValue` class is that we do not concatenate the "sha-256:" prefix in multiple places in code.